apps/server/src/toasts/toasts.service.ts

```diff
@@ -66,4 +79,32 @@ export class ToastsService {
       await toast.destroy();
     }
   }
+  async countHappenedToastsInLastSixMonths(): Promise<number> {
+    const currentDate = new Date();
+    const { startDate } = this.getCurrentPeriod(currentDate);
+    return await this.toastModel.count({
+      where: {
+        date: {
+          [Op.gte]: startDate,
+          [Op.lte]: new Date(
+            startDate.getFullYear(),
+            startDate.getMonth() + 6,
```

Comment:
This `+ 6` would be unclear to someone not familiar with the logic. it's best to add a constant that tells the reader what that means